ci-operator/jobs/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14-periodics.yaml

```diff
@@ -1846,17 +1846,6 @@ periodics:
     ci.openshift.io/generator: prowgen
     pj-rehearse.openshift.io/can-be-rehearsed: "true"
   name: periodic-ci-openshift-openshift-tests-private-release-4.14-amd64-nightly-4.14-upgrade-from-stable-4.13-aws-ipi-ovn-hypershift-replace-f7
-  reporter_config:
-    slack:
-      channel: '#forum-prow-hypershift-qe-ci'
```

Comment:
Unexpected changes ?

Reply:
@liangxia It's generated by executing `make update` automatically.

Reply:
Updated with generating by executing make update. @liangxia